Dockerfile

```diff
@@ -28,6 +28,27 @@ RUN apt-get update && apt-get install -y --no-install-recommends \
 # Used for platform-specific instructions
 ARG TARGETPLATFORM
 
+# cvxopt, an Augur dependency, does not have pre-built binaries for linux/arm64,
+# so it will have to be built on that platform.
+
+# Add system deps for building cvxopt
+RUN if [ "$TARGETPLATFORM" = "linux/arm64" ]; then \
+  apt-get install -y --no-install-recommends \
+      libblas-dev \
+      liblapack-dev \
+      libsuitesparse-dev \
+  ; \
+fi
+
+# "Install" (build) cvxopt separately since the process requires a special
+# environment variable.
+# https://github.com/cvxopt/cvxopt/issues/125#issuecomment-407396491
+RUN if [ "$TARGETPLATFORM" = "linux/arm64" ]; then \
+  CVXOPT_SUITESPARSE_INC_DIR=/usr/include/suitesparse \
+  pip3 install cvxopt \
+  ; \
+fi
```

Comment:
Maybe combine this into the previous RUN to avoid the two extra useless layers for non-arm64 images?

Reply:
Done in f60d84d.